cli/cmd/run.go

```diff
@@ -87,7 +90,20 @@ func buildRunStackCommand(key string, stack config.Stack) *cobra.Command {
 					"stack": stack.Name,
 				}
 				s := serviceManager.BuildFromConfig(srv)
-				serviceManager.Run(s)
+
+				if k == "kibana" {
+					s.SetWaitFor(wait.ForLog("http server running"))
```

Comment:
In metricbeat tests the healthcheck checks that kibana is green in `/api/status`.

Reply:
Testcontainers comes with a wait.ForHTTP("/api/status") that we could leverage instead of using the wait.ForLog("foo").

Reply:
Applied

Reply:
Ok, take into account that ideally we should wait till `/api/status` says that the state is green. Is it possible to extend `wait.ForHTTP()` to check the content of the response?